data/default_must_gather_versions.json

```diff
@@ -1,17 +1,22 @@
 {
 	"4.7": {
 		"cnv": "registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8:v2.6.5",
-		"ocs": "registry.redhat.io/ocs4/ocs-must-gather-rhel8",
-		"lso": "registry.redhat.io/openshift4/ose-local-storage-mustgather-rhel8"
+		"ocs": "registry.redhat.io/ocs4/ocs-must-gather-rhel8:v4.7",
+		"lso": "registry.redhat.io/openshift4/ose-local-storage-mustgather-rhel8:v4.7"
 	},
 	"4.8": {
 		"cnv": "registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8:v2.6.5",
-		"ocs": "registry.redhat.io/ocs4/ocs-must-gather-rhel8",
-		"lso": "registry.redhat.io/openshift4/ose-local-storage-mustgather-rhel8"
+		"ocs": "registry.redhat.io/ocs4/ocs-must-gather-rhel8:v4.8",
+		"lso": "registry.redhat.io/openshift4/ose-local-storage-mustgather-rhel8:v4.8"
 	},
 	"4.9": {
-		"cnv": "registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8:v2.6.5",
-		"ocs": "registry.redhat.io/ocs4/ocs-must-gather-rhel8",
-		"lso": "registry.redhat.io/openshift4/ose-local-storage-mustgather-rhel8"
+		"cnv": "registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8:v4.9.3",
+		"odf": "registry.redhat.io/odf4/ocs-must-gather-rhel8:v4.9",
+		"lso": "registry.redhat.io/openshift4/ose-local-storage-mustgather-rhel8:v4.9"
+	},
+	"4.10": {
+		"cnv": "registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8:v4.10.0",
```

Comment:
@tiraboschi please confirm whether the images are correct.

Reply:
+1

Reply:
ocs should be always present in the json. This is the default must gather image to use (not related to operators) and without  it the must-gather collection will not function propely

Also, why there are images missing in 4.10 If there are no new image release put there the versions from 4.9. We do not accumulate the versions so each entry should be a full description of the operator and ocs image inventory

Reply:
I have updated the images based on the version, for 4.10 I have used must gather operator images for 4.10